app/ROCrate/ROCrateGenerator.php

```diff
@@ -247,6 +303,54 @@ public static function generateCollectionCrate(Collection $collection, $path = n
                     $zip->addFromString($directory . '/' . self::getDatasetExportFileName($dataset, 'kml'), $dataset->kml());
                     $zip->addFromString($directory . '/' . self::getDatasetExportFileName($dataset, 'json'), $dataset->json());
                 }
+
+                //Add saved search files to zip
+                // Creating fake request to simulate an internal request to GazetteerController's search function
+                foreach ($collection->savedSearches as $savedSearch) {
+
+                    $directory = self::getSavedSearchDirectoryName($savedSearch);
+                    $zip->addEmptyDir($directory);
+                
+                    //Csv
+                    $url = url("/search" . $savedSearch->query . '&format=csvContent');
```

Comment:
This might be a bit tricky as '&format=csv' returns a streamed response instead of a string. Have tried output buffering but won't work. Maybe it is better to wait until we refactor the search function.

Reply:
Please add some comments stating the reason of using this new format.

Reply:
Updated